### DISCUSSION TITLE
What is the best PDF plugin to use with mkdocs-material?

### DISCUSSION THREAD

**Discussion by mudgen** [squidfunk/mkdocs-material#4315] [General]
There seems to be a few different PDF plugins that work with mkdocs.

Is there one that works best with mkdocs-material?

**Comment by dkebler** [comment]
year later....can anyone comment?   can mkdocs plugins work with mkdocs-material?

**Reply by squidfunk** [reply]
Yes, of course! [There are several solutions out there](https://github.com/mkdocs/catalog#-site-conversion-pdfepubetc). Which one is _the best_ may depend on your use case.

**Reply by dkebler** [reply]
Thx,   looks like this one is the only one with a recent (in 2023) commit and specifically says it will work with mkdocs-material.   Now to try it out....   

https://github.com/timvink/mkdocs-print-site-plugin
